wandb/sdk/artifacts/_utils.py

```diff
@@ -0,0 +1,40 @@
+"""Internal helper functions for artifacts."""
+
+from __future__ import annotations
+
+import re
+from typing import Collection
+
+
+def validate_aliases(aliases: Collection[str]) -> list[str]:
+    """Validate the given artifact aliases, returning them as a list if successful.
+
+    Raises:
+        ValueError: If any of the aliases contain invalid characters.
+    """
+    invalid_chars = ("/", ":")
+    if any(char in alias for alias in aliases for char in invalid_chars):
+        raise ValueError(
+            f"Aliases must not contain any of the following characters: {', '.join(invalid_chars)}"
+        )
+    return list(aliases)
```

Comment:
note: this is not really new code, just a new helper function from consolidating duplicated snippets from `artifact.py` and `wandb_run.py`